### ISSUE TITLE
Hyper-parameters for training Tiny-Imagenet

### ISSUE THREAD

**Issue by IISCAditayTripathi** [sjoshi804/eff-cl#4]
Hi,

I am training a ResNet18 model on the full TinyImagenet dataset with the hyperparameters given in the paper (Batch size 512, lr 0.001, num epochs 400), and I am getting the test accuracy of 23 % which is behind the reported numbers of ~32 %. Let me know if I am using the correct hyper-parameters or if any other change is required.

**Comment by sjoshi804** [comment]
Hi, thanks for reaching out! I believe actually I used hyper-parameters from previous papers that have worked for TinyImageNet here. I believe the learning rate should be 0.002 and I will get back to you on whether any other changes are required. In general though, since the focus of our method is the subset selection, we did not spend a lot of time tuning the hyperparameters for contrastive learning, but just ensured we use same hyperparamters for full data and any subset we try.

**Comment by IISCAditayTripathi** [comment]
I am still getting bad results in the TinyImagenet dataset. It gives ~35% on the training dataset and ~ 2% on the validation dataset. Does the TinyImagenet dataset require special preprocessing? Can you share your preprocessing for TinyIN and its hyperparameters?
